### PR TITLE
[T167308] Add translations to enable location explore feed card & add missing wmf_configureSubviewsForDynamicType

### PR DESCRIPTION
**Pull request by joewalsh**
- Adds the same translations used in places
- Adds missing `wmf_configureSubviewsForDynamicType`
- Uses the ability to override extension functions to eliminate the need for the type check in `wmf_configureSubviewsForDynamicType`

simulator screen shot jun 7 2017 11 20 36 am

### CODE REVIEW

**Comment by montehurd** [comment]
hmm that "South Wales" title and description looks like it's coming from enwiki...

**Comment by joewalsh** [comment]
@montehurd yes, the content is from enwiki but the system language is French

**Comment by joewalsh** [comment]
simulator screen shot jun 7 2017 1 18 48 pm

^ same screen but with French feed content

**Comment by montehurd** [comment]
> @montehurd yes, the content is from enwiki but the system language is French

oh so system lang was FR, but first search lang was EN?

**Comment by joewalsh** [comment]
yes

File: Wikipedia/Code/EnableLocationViewController.swift [hunk 1]
```diff
@@ -3,8 +3,12 @@ import UIKit
 protocol EnableLocationViewControllerDelegate: NSObjectProtocol {
     func enableLocationViewController(_ enableLocationViewController: EnableLocationViewController, didFinishWithShouldPromptForLocationAccess  shouldPromptForLocationAccess: Bool)
 }
-class EnableLocationViewController: UIViewController {
 
+class EnableLocationViewController: UIViewController {
+    static let localizedEnableLocationTitle = WMFLocalizedString("places-enable-location-title", value:"Explore articles near your location by enabling Location Access", comment:"Explains that you can explore articles near you by enabling location access. \"Location\" should be the same term, which is used in the device settings, under \"Privacy\".")
```
Comment (montehurd) [inline Wikipedia/Code/EnableLocationViewController.swift]: > \"Location\" should be the same term, which is used in the device settings

hehe guessing most translators aren't iOS users ;)

**Pull request merged**